esmvaltool/recipes/ref/recipe_ref_ozone.yml

```diff
@@ -0,0 +1,267 @@
+      start_longitude: 0
+      end_longitude: 360
+    extract_month:
+      month: 10
+    convert_units:
+      units: DU
+
+  create_time_series_NH:
+    annual_statistics:
+      operator: mean
+    area_statistics:
+      operator: mean
+    extract_region:
+      start_latitude: 60
+      end_latitude: 90
+      start_longitude: 0
+      end_longitude: 360
+    extract_month:
+      month: 3
+    convert_units:
+      units: DU
+
+  create_lat_time_map:
+    regrid:
+      scheme: linear
+      target_grid: 5x5
+    regrid_time:
+      calendar: standard
+    zonal_statistics:
+      operator: mean
+    convert_units:
+      units: DU
+
+  create_lat_month_map:
+    regrid: 
+      scheme: linear
+      target_grid: 5x5
+    climate_statistics:
+      operator: mean
+      period: month
+    zonal_statistics:
+      operator: mean
+    convert_units:
+      units: DU
+
+  create_zonal_mean_profile:
+    zonal_statistics:
+      operator: mean
+    regrid:
+      scheme: linear
+      target_grid: 5x5
+    extract_levels:
+      levels: [15000, 10000, 7000, 5000, 3000, 2000, 1000, 500, 100]
+      scheme: linear_extrapolate
+      coordinate: air_pressure
+    climate_statistics:
+      operator: mean
+      period: full
+    convert_units:
+      units: ppm
+
+
+
+# Note: the following model is just an example
+datasets:
+  - {project: CMIP6, dataset: CNRM-ESM2-1, grid: gr, ensemble: r1i1p1f2, exp: historical, alias: CNRM-ESM2-1}
+
+timerange_for_toz_monthly_data: &time_period_toz_monthly
+  timerange: '1950/2014'  # can be specified, this is just an example
+timerange_for_toz_zonal_mean: &time_period_toz_zonalmean
+  timerange: '1997/2014'  # is chosen to start with the start of the obs data
+timerange_for_ozone_profile: &time_period_o3_profile
+  timerange: '1990/2000'  # can be specified, this is just an example
+
+
+diagnostics:
+# Total column ozone plots
+  polar_cap_time_series_SH:
+    variables:
+      toz:
+        mip: AERmon
+        exp: historical
+        preprocessor: create_time_series_SH
+        <<: *time_period_toz_monthly
+    additional_datasets:
+      - {project: OBS6, dataset: ESACCI-OZONE, type: sat, version: L3, tier: 2, mip: AERmon, timerange: '1997/2022', alias: ESACCI-OZONE}
+    scripts:
+      plot:
+        script: monitor/multi_datasets.py
+        plot_folder: '{plot_dir}'
+        plot_filename: '{plot_type}_{real_name}_SH_Oct'
+        facet_used_for_labels: alias
+        savefig_kwargs:
+          dpi: 150
+          bbox_inches: tight
+          orientation: landscape
+        plots:
+          timeseries:
+            annual_mean_kwargs: false
+            plot_kwargs:
+              OBS:
+                color: black
+              CMIP6_CNRM-ESM2-1:
+                color: cyan
+              CMIP6_CNRM-ESM2-1:
+                color: cornflowerblue
+              CMIP6_UKESM1-0-LL:
+                color: gold
+              CMIP6_UKESM1-0-LL:
+                color: darkorange
```

Comment:
```suggestion
```

To make it usable for the REF with arbitrary datasets

Reply:
Well spotted!